controllers/pipeline/pipeline_controller_test.go

```diff
@@ -97,6 +97,7 @@ var _ = Describe("PipelineController", func() {
 					"pipelines.openshift.io/runtime":        "nodejs",
 					"pipelines.openshift.io/strategy":       "s2i",
 					"appstudio.openshift.io/component":      "component-sample",
+					"appstudio.openshift.io/application":    applicationName,
```

Comment:
I don't know if I can add this to the current pipeline, must it be separate pipeline ?

Reply:
This interpretation is fine.